build.fsx

```diff
@@ -368,25 +369,27 @@ module Nuget =
         match project with
         | "Akka" -> []
         | "Akka.Cluster" -> ["Akka.Remote", release.NugetVersion]
+        | "Akka.Cluster.TestKit" -> ["Akka.Remote.TestKit", release.NugetVersion; "Akka.Cluster", release.NugetVersion]
         | "Akka.Cluster.Sharding" -> ["Akka.Cluster.Tools", preReleaseVersion; "Akka.Persistence", preReleaseVersion]
-        | "Akka.Cluster.Tools" -> ["Akka.Cluster", preReleaseVersion]
+        | "Akka.Cluster.Tools" -> ["Akka.Cluster", release.NugetVersion]
+        | "Akka.MultiNodeTestRunner" -> [] // all binaries for the multinodetest runner have to be included locally
```

Comment:
Because the MNTR is an EXE, all of the dlls it needs to run must be available locally in its working directory. It takes no NuGet package dependencies. Everything it needs to execute is packaged into its `.nupkg`.